### ISSUE TITLE
Sanitizing unsafe style

### ISSUE THREAD

**Issue by davidpvilaca** [Mawi137/ngx-image-cropper#32]
I'm upgrade from version 0.1.24 to 0.1.30 then is showing angular warning:

![print](https://i.imgur.com/lbSxL6A.png "print")

```text
WARNING: sanitizing unsafe style value 0px; (see http://g.co/ng/security#xss).
```

on `platform-browser.js`

as the angular itself recommends: **see http://g.co/ng/security#xss**

**Comment by Mawi137** [comment]
Thanks, I'll look into it!

**Comment by Mawi137** [comment]
Should be fixed in 0.2.2!
